app/models/spree/line_item_decorator.rb

```diff
@@ -127,6 +127,12 @@ def sufficient_stock?
 
   private
 
+  def update_inventory_with_scoping
+    scoper.scope(variant)
+    update_inventory_without_scoping
+  end
+  alias_method_chain :update_inventory, :scoping
```

Comment:
are you saying that on alias_method_chain we should call super instead of <method>_without_<modification> (update_inventory_without_scoping)? I dont think so, all uses and examples I see make use of the "without" call.

Reply:
Sorry I was on my phone and I couldn't share a suggestion. What I meant was

```ruby
def update_inventory                                                                                                                                                                                         
 scoper.scope(variant)                                                                                                                                                                                      
 super                                                                                                                                                                                                      
end   
```

But this returns `NoMethodError: super: no superclass method `update_inventory' for #<Spree::LineItem:0x0055cc5b3a0610>`. Thanks `class_eval`... :sob: